stdlib/string.gr

```diff
@@ -872,6 +872,239 @@ export let endsWith = (search: String, string: String) => {
   }
 }
 
+/**
+ * Replaces the first appearance of the search pattern in the string with the replacement value.
+ *
+ * @param searchPattern: The string to replace
+ * @param replacement: The string to replace with
+ * @param string: The string to replace on
```

Comment:
Don't end these with `with` and `on`. Something like `The replacement` is probably enough and then `The string to change` seems fine